tests/virtual_machines_test.go

```diff
@@ -627,6 +598,26 @@ func newNetwork(name string) kubevirtv1.Network {
 	}
 }
 
+// This function allocates vms with 1 NIC each, in order to fill the mac pool to the brim.
+func AllocateVmsUntilFull(macsToLeaveFree uint64) error {
```

Comment:
Well the name is not always true since we pass `macsToLeaveFree`

Reply:
ack